osmosis-labs/osmosis-frontend
# Publish Stage

* Fix issue with token select in create pool workflow
* Swap router: filter by TVL instead of specific tokens

## Testing
* Make sure create pool token select works
* Check for performance concerns in swap tool, drawers. Especially on frontier.
* Double check swap tool on pool detail pages
